service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -78,6 +91,17 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
             .build();
     }
 
+    private String getHearingDate(List<Element<HearingBooking>> element) {
+        String date;
+
+        if (element.get(0).getValue().getStartDate() != null) {
```

Comment:
can we be sure that collection is properly ordered, I mean that first element is before second? Is it expected behaviour when hearing already happened (first element is in past)? 
Maybe existing HearingBookingService is better place for this method.

Reply:
hmm, so collection ordering I cannot guarantee. I guess this method should be refactored to get the earliest hearing. The first hearing is ALWAYS for the SDO. But you're right, just saying the first element could break things if the user was to use the collection screen differently to what I expect. I shall refactor.

Reply:
getMostUrgentHearingBooking seems to be doing this 👍